  <launch>
    <include file="$(find victor_perception)/launch/M3800-test.launch">
      <!--<arg name="cmd_vel_output" value="velocity_smoother_key_teleop/raw_cmd_vel" /> -->
    </include>
    
    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/color_tracker/output_image_raw"/>
    <param name="autosize" value="true" />
    </node> 
    
    <node name="color_tracker_controller" pkg="victor_perception" type="color_tracker_controller" respawn="false" output="screen">
    <remap from="/color_tracker/input_image_raw" to="/usb_cam/image_raw"/>
    </node> 
  </launch>